runtime/moonbase/src/lib.rs

```diff
@@ -1581,6 +1745,12 @@ impl Contains<Call> for NormalFilter {
 				pallet_assets::Call::cancel_approval { .. } => true,
 				_ => false,
 			},
+			// We want to disable create, as we dont want users to be choosing the
+			// assetId of their choice
+			Call::LocalAssets(method) => match method {
+				pallet_assets::Call::create { .. } => false,
```

Comment:
do we also need to disable `mint` like the call filter above?
```suggestion
				pallet_assets::Call::create { .. } => false,
				pallet_assets::Call::mint { .. } => false,
```

Reply:
never mind, I see this intentional